.travis.yml

```diff
@@ -68,6 +69,20 @@ x_base_steps:
     services:
       - docker
 
+  # Manifest list deploy job
+  - &manifest-deploy
+    stage: deploy-manifest
+    if: type != pull_request AND ( tag IS present OR branch = master OR commit_message =~ /\[travis deploy\]/ )
+    before_script:
+      - git config remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
+      - git fetch --unshallow --tags
```

Comment:
After this line as the last command in the `before_script` section, can you add the following?

```

      # TODO: remove the chmod command when docker-ce on travis is upgraded to 18.09.0+.
      # We need /etc/docker to be accessible to non-root users.
      # See https://github.com/moby/moby/pull/37847.
      - sudo chmod 0755 /etc/docker
```

Reply:
added!